src/translations/en.json

```diff
@@ -839,7 +839,9 @@
               "input_datetime": "Reload input date times",
               "input_select": "Reload input selects",
               "template": "Reload template entities",
-              "universal": "Reload universal media player entities"
+              "universal": "Reload universal media player entities",
+              "rest": "Reload rest entities",
+              "command_line": "Reload command_line entities"
```

Comment:
is `command_line` a unique name (with the underscore) or should it be `command line entities`

Reply:
We call them Command line here (https://www.home-assistant.io/integrations/sensor.command_line/) so probably "command line entities" makes sense  👍